components/progress-bar/README.md

```diff
@@ -45,31 +45,50 @@ to resolve the modules from the node_modules folder.
 
 # Configuration and Usage
 
-- The progress-bar has `data-animation-duration` attribute by which the animation
-duration between the start and the end target value is set. The value is
-expected to be a number and it is used as milliseconds.
+## Attributes
+
+### animation-duration
+
+The progress-bar has `animation-duration` attribute by which the animation
+duration between the start and the end target value is set. The value must be a positive number and it is used as milliseconds.
 
 The attribute is optional and if not provided, there will be no animation when
 setting the new progress.
 
 Here is an example:
 ```html
-<gameface-progress-bar
-  data-animation-duration="2000">
-</gameface-progress-bar>
+<gameface-progress-bar animation-duration="2000"></gameface-progress-bar>
 ```
 
-You can also set it using the JS:
+You can update it using the [setAttribute method](https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute):
+
+```js
+document.querySelector('gameface-progress-bar').setAttribute('animation-duration', 1000);
+```
+
+You can also set it using JS:
```

Comment:
Mention here that the animation will be restarted from the beginning if its duration is changed with from the progress-bar attribute because this is important IMO